src/libraries/System.Private.CoreLib/src/System/IO/PersistedFiles.Browser.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.IO
+{
+    internal static partial class PersistedFiles
+    {
+        internal static string GetHomeDirectory()
+        {
+            string? home = Environment.GetEnvironmentVariable("HOME");
+            return string.IsNullOrEmpty(home) ? "/" : home;
+        }
```

Comment:
could be simplified since the GetHomeDirectory mentions: https://github.com/dotnet/runtime/blob/d7506e7d04b5dadf40e48896a8c1585ed10cd354/src/libraries/System.Private.CoreLib/src/System/IO/PersistedFiles.Unix.cs#L82

and if we return null we _already_ have a fallback to using `/` as the homedir here:
https://github.com/dotnet/runtime/blob/d7506e7d04b5dadf40e48896a8c1585ed10cd354/src/libraries/System.Private.CoreLib/src/System/Environment.GetFolderPathCore.Unix.cs#L82-L88

```suggestion
        internal static string? GetHomeDirectory()
        {
            return Environment.GetEnvironmentVariable("HOME");
        }
```